libs/elements/src/lib/side-navigation/vl-side-navigation-toggle.element.ts

```diff
@@ -1,4 +1,4 @@
-import { BaseElementOfType, webComponent } from '@domg-wc/common-utilities';
+import {BaseElementOfType, findDeepestElementThroughShadowRoot, webComponent} from '@domg-wc/common-utilities';
```

Comment:
formattering is niet ingesteld op prettier lijkt het, maar dat is niet zo erg

Reply:
Klopt, enkel op eslint. Maar nu dus wel ;)